xml/obs_build_config.xml

```diff
@@ -348,6 +348,20 @@ Keep: debbuild</screen>
                       add an exclamation mark (<literal>!</literal>) as prefix.</para>
         </listitem>
        </varlistentry>
+       <varlistentry>
+        <term><parameter>ignoreconflicts</parameter></term>
+        <listitem>
+                <para>Ignore defined conflicts of packages. By default these are reported as unresolvable.
+                      </para>
+        </listitem>
+       </varlistentry>
+       <varlistentry>
+        <term><parameter>keepfilerequires</parameter></term>
+        <listitem>
+                <para>Dependencies to filenames get ignored by default. This switch can be used
```

Comment:
This is wrong in two ways:

- Direct file dependencies are never ignored, only indirect ones
- If FileProvides are specified, they are not ignored for indirect ones either

I suggest something like

> Dependencies on files are only fulfilled if matching FileProvides are specified in the project configuration. If those are missing, the dependency results in an "unresolvable" state for directly required files or in silent breaking of the dependency for indirectly required files. With this option, all file requires are honoured by default and lead to "unresolvable" if there are no matching FileProvides defined.

Reply:
File requires of "Requires:" tag are ignored by default, only BuildRequires: to files are not.

But I am fine with your text as well, will change it.

Reply:
I prefer "direct" and "indirect" because it also makes sense outside of RPM, e.g. in kiwi or Dockerfile contexts

Reply:
there you have actually another dimenson, direct and indirect packages for build env and notinstalled package sets.